app/main.py

```diff
@@ -1,6 +1,18 @@
-from typing import Callable
+from typing import Callable, Any
+from functools import wraps
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    seve_res = {}
+
+    @wraps(func)
+    def wrapper(*args, **kwargs) -> Any:
+        if args in seve_res:
+            print("Getting from cache")
+            res = seve_res[args]
```

Comment:
you must use `args` + `kwargs` as the keys of `seve_res`


also `seve_res` and `res` is  not clear variable names